src/Apps/Artist/Routes/AuctionResults/__tests__/AuctionResults.test.tsx

```diff
@@ -1,47 +1,33 @@
 import { AuctionResultsFixture } from "Apps/__test__/Fixtures/Artist/Routes/AuctionResultsFixture"
 import { AuctionResultsRouteFragmentContainer as AuctionResultsRoute } from "Apps/Artist/Routes/AuctionResults"
-import { MockBoot, renderRelayTree } from "DevTools"
-import { ReactWrapper } from "enzyme"
+import { MockBoot } from "DevTools"
+import { RelayStubProvider } from "DevTools/RelayStubProvider"
+import { mount, ReactWrapper } from "enzyme"
 import React from "react"
-import { graphql } from "react-relay"
 import { Breakpoint } from "Utils/Responsive"
 
-jest.unmock("react-relay")
-
 describe("AuctionResults", () => {
   let wrapper: ReactWrapper
 
-  const getWrapper = async (breakpoint: Breakpoint = "xl") => {
-    return await renderRelayTree({
-      Component: AuctionResultsRoute,
-      query: graphql`
-        query AuctionResults_Test_Query($artistID: String!) {
```

Comment:
Another thing about using `RelayStubProvider` is we don't pollute our `__generated__` folder with a bunch of test-related queries, and also don't need to run `yarn relay` while writing tests.